src/json-editor.component.scss

```diff
@@ -564,4 +564,12 @@ complex-list-field {
   add-field-dropdown {
     display: none;
   }
+}
+
+// Make nav tabs sticky
+.nav-tabs{
+    position: sticky;
+    top: 0;
+    background: white;
+    z-index: 1;
```

Comment:
do we need the `z-index`?

Reply:
Without it Text from editor is overlapping with this bar
![editor_example](https://user-images.githubusercontent.com/41995670/92581252-f21cdb80-f27e-11ea-832a-32afb4241f36.png)